lib/mwUtil.js

```diff
@@ -33,6 +33,15 @@ mwUtil.makeETag = (rev, tid, suffix) => {
  * Normalizes the request.params.title and returns it back
  */
 mwUtil.normalizeTitle = (hyper, req, title) =>  mwUtil.getSiteInfo(hyper, req)
+.catch((e) => {
```

Comment:
> @gwicke Em... It's a MW specific RB module, how would the domain be not controlled by RB?

For example, if we fetched the siteinfo for a domain passed in as a path parameter, on some wikimedia.org end point. To my knowledge this is not the case right now & does not seem very likely, so I'm happy to make the assumption that the domain is validated.